apps/explorer/forms.py

```diff
@@ -83,6 +83,23 @@ class PixelSetSelectForm(PixelSetExportForm):
     pass
 
 
+class SessionPixelSetSelectForm(forms.Form):
+
+    pixel_set = forms.ChoiceField(
+        choices=[],
+        error_messages={
+            'required': _('You must select one selected Pixel Set.'),
```

Comment:
What does that mean exactly? "select a selected pixel set" 😋

Reply:
Yeah you're right. What about: "You must pick on pixel set from stored selection"

Reply:
That's the error you get when trying to remove a non-existent pixel set right? Maybe we could be more generic since no user should really hit this case in reality

Reply:
Indeed.